x/records/module_ibc.go

```diff
@@ -78,7 +80,7 @@ func (im IBCModule) OnChanOpenTry(
 ) (string, error) {
 	// doCustomLogic()
 	// core/04-channel/types contains a helper function to split middleware and underlying app version
-	_, cpAppVersion := channeltypes.SplitChannelVersion(counterpartyVersion)
+	_, cpAppVersion := SplitChannelVersion(counterpartyVersion)
```

Comment:
I'm not positive but I think we pass don't need to call this and instead pass `counterpartyVersion` through to `im.app.OnChanOpenTry`

Definitely double check me on this though

Reply:
Oh I see, thank u